python/ray/_private/usage/usage_lib.py

```diff
@@ -144,6 +148,16 @@ def _generate_cluster_metadata():
     return metadata
 
 
+def print_usage_stats_heads_up_message() -> None:
+    if _usage_stats_prompt_disabled() or _usage_stats_enabled():
+        return
+
+    print(
+        "Usage stats collection will be enabled by default in the next release. "
+        "See https://docs.google.com/document/d/1ZT-l9YbGHh-iWRUC91jS-ssQ5Qe2UQ43Lsoc1edCalc/edit#heading=h.17dss3b9evbj for more details"
```

Comment:
Shorten this link?

Reply:
yes, I want to.

Several options:
1. Use tinyurl
2. Create a ray.io/usage-stats url that redirects to the google doc
3. Create a ray.io/usage-stats page that formally describes the usage stats ray is collecting, privacy policy, etc

I think eventually we need 3 but for now probably 1 or 2 is enough.

Reply:
+1 for option (3); as inspiration, it might be useful to refer to the PyPI and conda package stats privacy policy pages:

* https://www.python.org/privacy/
* https://www.anaconda.com/privacy-policy

Reply:
2 Sounds good to me for the short term solution. (tiny URL seems a bit weird and less professional IMO)